.gitignore

```diff
@@ -1 +1,4 @@
 Godeps/_workspace
+.DS_Store
+testpop
```

Comment:
`testpop` shows up for me in the root of the project when build with `make testpop`. Hence I added it to this `.gitignore`. Does it show up in `scripts/testpop` for you?